FWCore/Framework/src/ScheduleItems.cc

```diff
@@ -141,6 +144,7 @@ namespace edm {
                      *preg_,
                      *branchIDListHelper_,
                      *thinnedAssociationsHelper_,
+                     subProcessParentageHelper_ ? &*subProcessParentageHelper_ : nullptr,
```

Comment:
probably simpler to do `subProcessParentageHelper_ ? subProcessParentageHelper_.get() : nullptr`